src/store/utils.js

```diff
@@ -33,4 +34,29 @@ export async function stacRequest(cx, link) {
     opts = link;
   }
   return await axios(opts);
-}
\ No newline at end of file
+}
+
+export const convertCoordinatesToEpsg4326 = (stacObj) => {
+    if (stacObj && !!stacObj['crs']) {
+      const stacEpsg = stacObj.crs.init.toUpperCase();
+
+      if (!!stacObj['geometry'] && stacObj['type'] === 'Feature') {
+        const geoCoordinates = stacObj.geometry.coordinates[0];
+        const coordinatesConverted = geoCoordinates.map((elem) => {
+          return proj4(`${stacEpsg}`, "EPSG:4326", [elem[0], elem[1]]);
+        });
+        
+        stacObj.geometry.coordinates[0] = coordinatesConverted;
```

Comment:
Our discussion about the bbox in the `Collection` case, reminded me that here we should also add the transformation of the bbox. As it is now, you are transforming only the `geometry`, but not the `bbox` parameter of the `Item`.

Reply:
Definitely, i will perform here the multilevel array update as well.